fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java

```diff
@@ -22,14 +18,7 @@
 import org.fakekoji.model.Platform;
 import org.fakekoji.storage.StorageException;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
same, change settings in idea

Reply:
Just tell me how. I hate that when idea is doing that. I had double check taht every "optimize imports" or simialr are disabled. Still it keeps doing that after the number of exports form some package exeeed some length:"(

Reply:
Settings -> Code Style -> Java -> Imports -> 
Use single class import - checked
Class count ... -  99
Names count .... - 99
work for me